### ISSUE TITLE
Looking to streamline the digital interview process at ATD. Currently we create interview question f...

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#11621]
> Name

Jennifer Watts 

> What application are you using?

Other / Not Sure

> Describe the problem.

Looking to streamline the digital interview process at ATD. Currently we create interview question files in word, convert to Adobe, create a fillable form, email to the panel with the hope they can open the files and their computer is compatible with the documents, then we need to get the files emailed back from the panel member before we can move to the next step in the hiring process. After the interviews, if we get a PIR (which is very often), we have to open flatten each file, combine the files, then complete the PIR. All together this is a timely and "busy work" process. 

> Describe the solution you have in mind.

With your expert advice I would like to see the interview question files be created and stored in Knack. The complete "Hiring Packet" could possibly be stored in Knack as well. Open to options to evolve our process. As of March 3/2023 HR has completed 78 recruitments this FY. This solution will affect the department as a whole. 

> How will we know that our solution is successful?

We would measure success by eliminating time spent from HR Advisors on creating each interview question sheet, the advisor not having to spend up to 4 hours to format each PIR, minimize the time spent on waiting for question files to be returned from the panel will allow HR to provide offer information to the hiring manager, thus more of a chance we are able to bring in the top talent ATD desperately needs. 

> Users

HR will manage, all ATD will benefit. 

> Sponsorship

Anthony Segura

> Which Strategic Direction 2023 “Mobility” indicators would this new application impact?

['System efficiency and congestion']

> Which Austin Strategic Mobility Plan goals would this new application support?

['Commuter Delay – Reduce the amount of time workers spend traveling between home and work.', 'Travel Choice — Promote a balanced transportation network and the ability to make informed choices based on personal needs and preferences.', 'Health & Safety — Protect Austinites by lowering the risk of travel-related injury and promoting public health.', 'Affordability — Lower the cost of traveling in Austin by providing affordable travel options.', 'Sustainability — Promote integrated designs and quality additions to the built environment while reducing impacts and promoting efficient use of public resources.', 'Placemaking — Build a transportation network that encourages social interaction through quality urban design, and connects users to the many places that make Austin unique.', 'Economic Prosperity — Promote economic growth for individuals and the city through strategic investments in transportation networks that meet the needs of the 21st century.', 'Innovation — Draw inspiration from forward-looking cities around the world, change the way we think about what’s possible, and set an example for the rest of the country.']

> Describe any workarounds you currently have in place or alternative solutions you've considered.

In 2020 we created the digital process. We are currently doing the best we can without a further digital process. 

> Is there anything else we should know?

By being able to recruit faster and have offers in hands of hiring managers more swiftly, this project has the ability to positively affect all aspects of the ASMP. 

> How have other divisions/departments/cities addressed similar challenges? 

After the SSPR conversion to the Knack system was such a success, I feel this process will operate simmilar. 

> Requested By
Jennifer W.

Request ID: DTS23-106580

**Comment by dianamartin** [comment]
Christina and I had a conversation about this after she had an HR check-in meeting with Jennifer. We ended up creating a Miro [board](https://miro.com/app/board/uXjVPpGwiwA=/) for the HR Interview Management.

**Comment by ChristinaTremel** [comment]
Scheduled meeting with Jennifer and Diana to discuss this project.

**Comment by ChristinaTremel** [comment]
# 5/8
- When they receive questions from hiring manager, they add them to a template in word and convert to PDF, then create fillable PDF. They then file each one down for the interviewer and the applicants and zip them in a file
- **Pain Points:**
   - Making files fillable
   - Emailing them out and getting them back via email
   - They get PIRs very often, so when that happens they have to open each file, flatten the form, and merge into one file to submit
   - The time it takes to create the documents is time consuming
- Public Works might have something that actually works, but there are also some issues
   - We build the application file in teams, but they keep question files per panel member there too. The questions are in there, you open in the app, and it shares via SharePoint.
   - This makes it faster in some ways, but for a PIR there would still be many documents to look at.
   - A lot of people don't understand how to open it in the app
   - The Public Works form doesn't calculate like our form does
- **Structure**:
   - Required fields:
      - Interview answers
      - Interview scores
      - Click here to sign and submit with an HR disclosure
   - Required documents:
      - Would we want to load applications and resumes in Knack or on a Teams channel? They currently just delete the teams channel with the applications and resume. **Decision**: for now we're going to keep the files in Teams.
   - **Notifications**:
      - One HR gets things set up, the day before the interview or that morning, HR would send a link with the access code to their page. Jennifer doesn't know if they should send it or if it should come from the system.
      - Once panel members submit, send an email to the person that created the file
      - Doesn't think we need a reminder message
 - There is something that was purchased by Corporate HR and she imagines that the product will help with the hiring process, but not with the actual interviewing process. 
 - Panel members, 9.5 out of 10 are department employees. Would having the app in the HR app help with creating the interviewing items. 
- Diana asked about if we should have this within the HR Knack app or outside of it:
   - Christina suggested possibly having the interview app outside for the HR application so we can lock it down more.
- **Timeline**:
   - Would be cool to have something set before the initial merge and we don't know when that is before October 1st.  
   - We might have more information on that May 19th.
- **Value of this Project:**
   - Amount of time Tina spends on setting up the interview with questions, documents, and etc: takes a couple of hours and but this is when she's fully focused/can't do anything else. Typically she gets interrupted while she's doing this.
   - When the interview is over, HR has to gather all this information together
   - PIRs also take a couple hours to pull all this information together
   - 4 interrupted man hours is what it takes
   - ATD does about 150 interview panels a year (this is just per PCN and many positions have multiple rounds
   - Will be adding about 500 employees with the merge with PWD (not include temps), so we will probably triple the amount of interviews we do next year

**Comment by dianamartin** [comment]
# 5/8 - Diana's Notes

**Comment by ChristinaTremel** [comment]
# 3/27 Notes - things to consider
- Assessments can be weighted differently 
- Seniority Points
- MCS rules state that all interview scores average to 100 points
   - Check the Scoring Matrix given by HR
